### ISSUE TITLE
Internet Explorer Promise is not Defined

### ISSUE THREAD

**Issue by rlsilv** [lukechilds/merge-images#96]
Hello,
Sorry for my ignorance, but does your "merge-images" also work in Internet Explorer?
I'm asking this because I'm having the error(Promise is not defined) when merging the images.
 
```
var arrayImages = [];
      for (var i = 0; i < boundsheight; i += imgHeight) {
        for (var j = 0; j < boundswidth; j += imgWidth) {
          img = myDiagram.makeImage({
            scale: 1,
            background: "white",
            position: new go.Point(p.x + j, p.y + i),
            size: new go.Size(imgWidth, imgHeight),
            type: "image/png"
          });
          arrayImages.push({src:img.src,x:p.x + j,y:p.y + i});
        }
      }
      console.log(arrayImages);
      mergeImages(arrayImages).then(function(b64){ document.querySelector('img').src = b64});
```

**Comment by lukechilds** [comment]
The library uses promises, so if you want it to run in a browser that doesn't support the Promise API then you'll need to load a Promise polyfill.

**Comment by rlsilv** [comment]
Hello,
I downloaded the promise polyfill bookstore which fixed the problem of Promise is not defined, but this generated another error that Object does not support the assign method. I also fixed this, but now I have the error  "Possible Unhandled Promise Rejection: Error: Couldn't load image".
It works in Firefox, Chrome but not in IE. I have been searching and it is related to the loading of base 64 images in IE. Any way to counter this problem?
